### DISCUSSION TITLE
How to Build Android and iOS nuget packages

### DISCUSSION THREAD

**Discussion by Agredo** [microsoft/onnxruntime-genai#1079] [Q&A] [answered]
Hi would like to test my .NET MAUI implementation (works quite nicely on Windows) on Android and iOS, but I don't know if it is possible to build Android and iOS Nuget packages. Is this currently possible? 

If so, how? 

Thank you all very much. I'm really looking forward to using ONNX runtimegenai for mobile devices.

**Comment by scsonic** [comment]
android: aar
iOS: framework
flutter: package

**Reply by Agredo** [reply]
But they are not Nuget Packages. It seems like there is support for them? https://github.com/microsoft/onnxruntime-genai/pull/923

But how do I built them?

**Comment by RyanUnderhill** [comment, marked as answer]
@skyline75489 Is this the same as issue #1081 ?
